### PR TITLE
docs(readme): add syntax highlighting to code examples

### PR DESCRIPTION
**Pull request by flootr**
• add syntax highlighting to code examples
• add `$` to npm cmd to indicate shell usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ Gridicon renders a single svg icon based on an `icon` prop. It takes a size prop
...
+```sh
+$ npm install gridicons --save
 ```
 
 #### Usage
 
 You can either import the whole iconset and decide at run-time which icon to use:
 
-```
+```js
```
Comment (nosolosw) [inline README.md]: Shall we use `jsx` instead?

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ render() {
 
 Or import icons individually:
 
-```
+```js
```
Comment (nosolosw) [inline README.md]: Same here.

File: README.md (hunk 1 above)
Comment (flootr) [reply README.md]: I didn't know that existed. TIL thanks! :)

File: README.md [hunk 3]
```diff
@@ -10,15 +10,15 @@ Gridicon renders a single svg icon based on an `icon` prop. It takes a size prop
 
 There's a gallery with all the available icons in http://automattic.github.io/gridicons/.
 
-```
-npm install gridicons --save
+```sh
+$ npm install gridicons --save
```
Comment (nosolosw) [inline README.md]: Perhaps we should not include the `$` bash mark? That makes difficult to copy-paste and I guess that highlighting it is bash code would be enough?

Comment (flootr) [reply README.md]: I agree that makes copying harder. Let's don't include it. Syntax highlighting doesn't give us much highlighting for shell commands so what about including a small comment that this is intended to be executed in a shell?

**Pull request merged**